### PR TITLE
telegraph (add telegraph in requirements if give module error) #test before merge 

### PR DESCRIPTION
**Pull request by New-dev0**

### CODE REVIEW

File: plugins/telegraph.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+from telegraph import Telegraph, upload_file, exceptions
+
+telegraph = Telegraph()
+r = telegraph.create_account(short_name="@EduuRobot on telegram")
+
+@Client.on_message(filters.command("telegraph", prefix))
+async def telegraph(c: Client, m: Message):
+    if m.reply_to_message:
+        if m.reply_to_message.photo or m.reply_to_message.video or m.reply_to_message.animation:
+            d_file = m.reply_to_message.download()
```
Comment (alissonlauffer) [inline plugins/telegraph.py:15]: There is a missing await in this line (15).

Comment (Sunda001) [reply plugins/telegraph.py:15]: just:
```if m.reply_to_message and m.reply_to_message.media:
  ...```

Comment (Sunda001) [reply plugins/telegraph.py:15]: rip markdown

Comment (Sunda001) [reply plugins/telegraph.py:15]: just: filters.media

Comment (iiiiii1wepfj) [reply plugins/telegraph.py:15]: > just: filters.media

yes, its better but without audio (its short than the code now)

Comment (New-dev0) [reply plugins/telegraph.py]: > just: filters.media
filters.media also include stickers and they will not be uploaded to telegraph simply
🤔

Comment (alissonlauffer) [reply plugins/telegraph.py]: Now it looks ok to me. Let's see what @Sunda001 thinks.

**Review by iiiiii1wepfj — COMMENTED** [review]
add support to text if you want

File: plugins/telegraph.py (hunk 1 above)
Comment (Sunda001) [reply plugins/telegraph.py]: i think its ok

**Comment by iiiiii1wepfj** [comment]
its ok @alissonlauffer ?

**Comment by alissonlauffer** [comment]
> its ok @alissonlauffer ?



> its ok @alissonlauffer ?

Now yes.

**Comment by alissonlauffer** [comment]
> > its ok @alissonlauffer ?
> 
> Now yes.

**Pull request merged**